src/qt/optionsdialog.cpp

```diff
@@ -302,6 +310,11 @@ void OptionsDialog::setMapper()
     mapper->addMapping(ui->proxyIpTor, OptionsModel::ProxyIPTor);
     mapper->addMapping(ui->proxyPortTor, OptionsModel::ProxyPortTor);
 
+    if (QSystemTrayIcon::isSystemTrayAvailable()) {
+        mapper->addMapping(ui->hideTrayIcon, OptionsModel::HideTrayIcon);
+        mapper->addMapping(ui->minimizeToTray, OptionsModel::MinimizeToTray);
+    }
+
```

Comment:
14228 I believe this is a problem. 

Should be done here: https://github.com/dashpay/dash/pull/4313/commits/d5372579def7ebce3281241f03e8582a7ee69bdd#diff-1d6f5264407b3b2e18be8d9d9653bbf1d67979f4f95dc26575dc23bb8463b4e7R280-R281 IE line 280-281